src/utils/convertBlobAttributes.ts

```diff
@@ -23,20 +23,23 @@ async function blobToBinary(blob: Blob | File): Promise<{
   dataBase64: string
   filename: string
 
-  // TODO: Remove for pisa
-  contentLength: number
-  contentType: string
-  _id: string
+  contentLength?: number // only needed for "localStorage" fake
+  contentType?: string // only needed for "localStorage" fake
+  _id?: string // only needed for "localStorage" fake
```

Comment:
* [ ] FYI Currently a single headline comment would do.
* [ ] FYI I'd not call it "fake" b/c there are valid use cases for local storage.